pages/survey-form/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta sharset="utf-8">
+    <meta name="viewport" content="device-width", initial-scale 1.0>
+    <title>SURVEY-FORM</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="icon.ico">
+  </head>
+
+  <body>
+
+    <div class="container">
+
+      <header>
+        <h1 class="center-text">freeCodeCamp Survey Form</h1>
```

Comment:
seems like both are center-text. I think it will work if you put the `center-text` class in the `header` instead. Try it.